rpm/udisks2.spec

```diff
@@ -1,35 +1,36 @@
-%global glib2_version                   2.36
+%global glib2_version                   2.50
 %global gobject_introspection_version   1.30.0
 %global polkit_version                  0.102
 %global systemd_version                 208
 %global dbus_version                    1.4.0
-%global libblockdev_version             2.14
+%global libblockdev_version             2.25
```

Comment:
Old thing but not sure if this version indirection provides much. Wouldn't mind just having the dependencies the normal way. These are rarely too interesting, e.g. above dbus 1.4.0 looks 2010 so it would be all the same even just requiring dbus without an exact version.